src/index.html

```diff
@@ -41,6 +41,13 @@
     <script type="text/javascript" src="bezier.js"></script>
     <script type="text/javascript" src="arrow.js"></script>
     <script type="text/javascript" src="ui.js"></script>
+    <!-- Web app manifest -->
+    <link rel="manifest" href="manifest.json">
+    <!-- Service worker -->
+    <script type="module">
+        import { Workbox } from "workbox/workbox-window.prod.mjs";
```

Comment:
```suggestion
        import { Workbox } from "./workbox/workbox-window.prod.mjs";
```
Without this, I get an error when I run this in localhost:
```
Uncaught TypeError: Failed to resolve module specifier "workbox/workbox-window.prod.mjs". Relative references must start with either "/", "./", or "../".
```

Reply:
Ah, my bad. Thanks for spotting this.